### PR TITLE
dockerfile: include binary and fix entrypoint

### PR DESCRIPTION
**Pull request by rphillips**
/cc @sjenning

### CODE REVIEW

**Comment by sjenning** [comment]
wow, I was having a bad day!
/lgtm

**Comment by rphillips** [comment]
np. I approved it :)

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
+FROM openshift/origin-release:golang-1.10 AS builder
 COPY . /go/src/github.com/openshift/pod-checkpointer-operator
 RUN cd /go/src/github.com/openshift/pod-checkpointer-operator && make build
 
 FROM centos:7
 COPY manifests /manifests
 
+COPY --from=builder /go/src/github.com/openshift/pod-checkpointer-operator/pod-checkpointer-operator /pod-checkpointer-operator
+
+ENTRYPOINT /pod-checkpointer-operator
```
Comment (sjenning) [inline Dockerfile:10]: don't really need the entrypoint as the command is dictate by the pod, but it doesn't hurt anything

**Comment by rphillips** [comment]
manually merging to fix the image

**Pull request merged**